render/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# This Dockerfile is based off the Google App Engine Python runtime image
+# https://github.com/GoogleCloudPlatform/python-runtime
+FROM gcr.io/google-appengine/python
+
+# Add metadata to Docker image
+LABEL maintainer="csse-education-research@canterbury.ac.nz"
+
+# Set terminal to be noninteractive
+ARG DEBIAN_FRONTEND=noninteractive
+ARG PROCESS_PORT=8080
+ARG FLASK_PRODUCTION=1
+
+ENV PORT ${PROCESS_PORT}
+ENV FLASK_PRODUCTION=${FLASK_PRODUCTION}
```

Comment:
Can you explain these steps here. I think I understand what is happening, but unsure why we don't do this for Django.